catalog/spatial/wfs/spatial-wfs-common/pom.xml

```diff
@@ -41,17 +41,17 @@
                                         <limit implementation="org.codice.jacoco.LenientLimit">
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.63</minimum>
```

Comment:
Note: instruction coverage did not actually decrease. On DDF master the instruction coverage is 0.62 but passes the build because of the lenient limit.